modules/mco-update-boot-images.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+// * nodes/nodes-nodes-managing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="mco-update-boot-images_{context}"]
+= Updating boot images
+
+A boot image is the image used by the Machine Config Operator (MCO) to boot up a {op-system-first} node. By default, the boot image is not managed by {product-title}. 
+
+This means that the boot image in your cluster is not updated along with your cluster. For example, if your cluster was originally created with {product-title} 4.11, the boot image used by the cluster is the same 4.11 version. When you scale up a new node, the MCO boots up that node with the originally-installed boot image and then automatically upgrades the node to the current cluster version. While this is a seamless process to the user, this process can cause issues such as the following conditions: 
+
+* Extra time to start up nodes
+* Certificate expiration issues
+* Version skew issues
+
+To avoid these issues, you can configure your cluster so that it updates the boot image anytime that you update your cluster. You can enable this feature by modifying the `MachineConfiguration` object. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters and is not supported for clusters managed by the Cluster CAPI Operator.
```

Comment:
```suggestion
To avoid these issues, you can configure your cluster so that it updates the boot image anytime that you update your cluster. You can enable this feature by modifying the `MachineConfiguration` object named `cluster`. Currently, the ability to update the boot image is available for only Google Cloud Platform (GCP) clusters and is not supported for clusters managed by the Cluster CAPI Operator.
```